docs/source/portal-molecule.rst

```diff
@@ -23,21 +23,68 @@ Molecules can be created via NumPy, Psi4-like molecule strings, and QCSchema JSO
     Molecules can automatically be orientated to a common frame of reference using the ``orient=True`` kwarg.
     This is useful for finding identical molecules.
 
+
+A full description of the various specifications can be found below:
+
+Required Keys
++++++++++++++
+
+   +-----------------------------------------------+--------------------------------------------------------------------------------------------------------------------------+-----------------------------+
+   | Key Name                                      | Description                                                                                                              | Field Type                  |
+   +===============================================+==========================================================================================================================+=============================+
+   | symbols                                       | (nat, ) atom symbols in title case.                                                                                      | array[string]               |
```

Comment:
Small nuance: but the "nat" took a bit for me to figure out it meant "N atom", I don't know if there is a better way to show this since "nat" is phonetically pronounceable, mabye "N-at?" Fine for now, something to think about.

Reply:
Add a note at the bottom